viewer/static/js/app/components/views/experiment-metadata.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default function(props) {
+  let maybeName;
```

Comment:
I'm glad you brought this up. As it turns out, the [React documentation](https://facebook.github.io/react/docs/conditional-rendering.html#element-variables) uses `null` as a default. So I'll do that.